src/tir/op/op.cc

```diff
@@ -653,7 +653,27 @@ PrimExpr abs(PrimExpr x, Span span) {
     if (px) {
       return IntImm(x.dtype(), std::abs(px->value), px->span);
     }
-    return tir::Select(x >= make_zero(x.dtype()), x, -x, span);
+
```

Comment:
Hmm honestly I'm not sure if I like this change since it actually is a change in the semantics of abs. 

E.g. abs(-128) = 128 which should overflow to -128 with only 8 bits. 

IMO two solutions:
1. For integers, make it return uint version so it never overflows.
2. Just handle the edge case manually in fq2i 

If I understand this is only important to handle the -128 --> 127 case in quantized models so I say just put it in 2.